examples/docker/config/start.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+pip3 install gunicorn &&\
```

Comment:
Easier to read if you don't have line continuations. This isn't a Dockerfile so there's no layer caching benefit or anything.